frontend/src/metabase/actions/components/ActionViz/Action.unit.spec.tsx

```diff
@@ -158,6 +158,8 @@ async function setup({
 
   // Wait until UI is ready
   await screen.findByRole("button");
+
+  return { view };
```

Comment:
not used?

Reply:
afaik we don't leave unused code just in case, cc @WiNloSt

Reply:
I agree with @ranquild. If it's easy enough to add that when writing/debugging new tests then let's not have it here. I'm worried we'll tend to leave certain stuff for purposes we might not even use again. And it'd be a lot simpler/cleaner to understand if we don't leave this variable unused. If I were to read this function I'd expect this `view` to be used somewhere in the test.